Update README.md (#10)
biomejs/biome-intellij: README.md

> # Biome - Intellij Plugin

Reviewer: ```suggestion
# Biome - IntelliJ Plugin
```

[![IntelliJ IDEA Ultimate Version](https://img.shields.io/badge/IntelliJ%20IDEA%20Ultimate-2023.2.2-374151.svg?labelcolor=000&logo=intellij-idea&labelColor=black)](https://www.jetbrains.com/idea/)
[![WebStorm Version](https://img.shields.io/badge/WebStorm-2023.2.2-1F3263.svg?labelcolor=000&logo=webstorm&labelColor=black)](https://www.jetbrains.com/webstorm/)
[![AppCode Version](https://img.shields.io/badge/AppCode-2023.2.2-2380B0.svg?labelcolor=000&logo=appcode&labelColor=black)](https://www.jetbrains.com/objc/)
[![PhpStorm Version](https://img.shields.io/badge/PhpStorm-2023.2.2-953D8C.svg?labelcolor=000&logo=phpstorm&labelColor=black)](https://www.jetbrains.com/phpstorm/)
[![RubyMine Version](https://img.shields.io/badge/RubyMine-2023.2.2-A11523.svg?labelcolor=000&logo=ruby&labelColor=black)](https://www.jetbrains.com/ruby/)

[Biome](https://biomejs.dev/) is a powerful tool designed to enhance your development experience. 

> This plugin integrates seamlessly with many [JetBrains IDE's](#supportedIDEs)to provide some capabilities:

Reviewer: ```suggestion
This plugin integrates seamlessly with many [JetBrains IDE's](#supported-ides) to provide the following capabilities.
```

> - 💡  See lints while you type

Reviewer: ```suggestion
- 💡 See lints while you type
```

> - 👨‍💻 Apply code fixes (from mouse-over, <kbd title="Option">⌥</kbd>+<kbd  title="Enter">⏎</kbd> or <kbd title="Alt">Alt</kbd>+<kbd title="Enter">Enter</kbd>)

Reviewer: ```suggestion
- 👨‍💻 Apply code fixes
```

Let's keep it simple and instead put it in the usage section.

- 🚧 Reformat your code

However, please note the following limitations:

> - 💾 Automatically applying code fixes on save

Reviewer 2: This will now be supported by the plugin, we can move it to the supported features list.

Can you also include "💾 Automatically format code on save"?

Author: Hi @victor-teles, I appreciate your suggestion. Feel free to suggest more changes with shortcuts or other new features . I've already implemented the automatic format code on save at feature list.

> - 🚧 Reformat your code with <kbd>⌥⇧</kbd>+<kbd title="Cmd">⌘</kbd>+<kbd  title="L">L</kbd> or <kbd title="Ctrl">Ctrl</kbd>+<kbd title="Alt">Alt</kbd>+<kbd  title="L">L</kbd> (You can also format your [code on save](https://www.jetbrains.com/help/webstorm/reformat-and-rearrange-code.html#reformat-on-save))

Reviewer: ```suggestion
- 🚧 Reformat your code
```

### From disk

1. Download the plugin .zip from releases tab.

> 2. Press <kbd>⌘Cmd</kbd>, to open the IDE settings and then select Plugins.

Reviewer: ```suggestion
2. Press <kbd>⌘Cmd + ,</kbd> to open the IDE settings and then select Plugins.
```

> 2. Press `⌘Сmd,` to open the IDE settings and then select Plugins.

Reviewer: I believe this could be improved with a `kbd` tag

Author: Right. I've incorporated the <kbd>kbd</kbd> tag to enhance the representation of the command:

Press <kbd>⌘Cmd</kbd>, to open the IDE settings and then select Plugins.

3. On the Plugins page, click The Settings button and then click Install Plugin from Disk….

## Getting Started
### Biome Resolution

> The Plugin tries to use Biome from your project’s local dependencies (`node_modules/.bin/biome`). We recommend adding Biome as a project dependency to ensure that NPM scripts and the extension use the same Biome version.

Reviewer: ```suggestion
The plugin tries to use Biome from your project’s local dependencies (`node_modules/.bin/biome`). We recommend adding Biome as a project dependency to ensure that NPM scripts and the extension use the same Biome version.
```

You can also explicitly specify the `Biome` binary the extension should use by configuring the `Biome CLI Path` in `Settings`->`Language & Frameworks`->`Biome Settings`.

### Plugin settings

#### `Biome CLI Path`

This setting overrides the Biome binary used by the plugin.

> ### <a name="supportedIDEs">Supported IDEs</a>

Reviewer: ```suggestion
### Supported IDEs
```

If the link to the header is `#supported-ides`, resorting to this trick is not necessary.
